3_postnatal_care.do

```diff
@@ -57,12 +57,10 @@ c_pnc_eff2_q
 			global pnc "a b i"
 		}
 		if inlist(country_name,"Tonga2019") {
-			global pnc "a b c d g"
+			global pnc "a"
```

Comment:
This is interesting here. It seems b c d g have been dropped in your latest version. We could discuss/walk through them when you have time.

Reply:
this is a bit like saving habit lol. first time I was puzzling about who can be considered as skilled workers; and I was mentioned this question in the wechat group. but I was 99.9% sure about doctor should be one of them. so I save and push the file. then it is clarified - we need to search for the answer ourself in the survey report. four other variables were added.

short answer is - "b c d g" should be added
b - nurse/midwife
c - health officer
d - community nurse
g - community health worker

Reply:
this is a bit like saving habit lol. first time I was puzzling about who can be considered as skilled workers; and I was mentioned this question in the wechat group. but I was 99.9% sure about doctor should be one of them. so I save and push the file. then it is clarified - we need to search for the answer ourself in the survey report. four other variables were added.

short answer is - "b c d g" should be added
b - nurse/midwife
c - health officer
d - community nurse
g - community health worker

Reply:
this is a bit like saving habit lol. first time I was puzzling about who can be considered as skilled workers; and I was mentioned this question in the wechat group. but I was 99.9% sure about doctor should be one of them. so I save and push the file. then it is clarified - we need to search for the answer ourself in the survey report. four other variables were added.

short answer is - "b c d g" should be added
b - nurse/midwife
c - health officer
d - community nurse
g - community health worker

Reply:
I see, reading this, I feel we might want to be loyal to the definition used in report. That means a,b,c,d; g is not explicitly referenced.

Reply:
checked

g - community health worker 
removed